elide-model-config/src/main/java/com/yahoo/elide/modelconfig/jsonformats/ElideNameFormatAttr.java

```diff
@@ -36,8 +40,12 @@ public void validate(final ProcessingReport report, final MessageBundle bundle,
                     throws ProcessingException {
         final String input = data.getInstance().getNode().textValue();
 
-        if (!input.matches(NAME_FORMAT_REGEX)) {
+        if (!NAME_FORMAT_REGEX.matcher(input).matches()) {
             report.error(newMsg(data, bundle, FORMAT_KEY).putArgument("value", input));
         }
+
+        if (input.equalsIgnoreCase("from")) {
```

Comment:
Why isn't from allowed?

Reply:
I thought it may conflict with `sqlhelper` syntax but since there is not conflict, removing it.